src/crawler.ts

```diff
@@ -2106,6 +2106,10 @@ self.__bx_behaviors.selectMainBehavior();
 
     let finished = false;
 
+    // disable extraHops for sitemap found URLs by setting to extraHops limit + 1
+    // otherwise, all sitemap found URLs would be eligible for additional hops
+    const extraHopsDisabled = this.params.extraHops + 1;
```

Comment:
I think those should still be found and added normally from link extraction, since they would just be ignored / not added to queue or seen list in the sitemap pass. Not disabling the extra hops has the effect of treating all the sitemap URLs as if they're found as links from the seed page, so they all get added!
I suppose could even disable sitemaps for non-sitewide scope, but I think this make sense for the time being..

Reply:
Ah good point!